webrtc.html

```diff
@@ -2625,6 +2687,194 @@
         </dl>
       </section>
 
+      <section>
+        <h3>RTCDtlsTransport Interface</h3>
+
+        <p>The <code>RTCDtlsTransport</code> interface allows an
+        application access to information about the Datagram Transport
+        Layer Security (DTLS) transport over which RTP and RTCP
+        packets are sent and received by <code>RTCRtpSender</code>
+        and <code>RTCRtpReceiver</code> objects, as well other data
+        such as SCTP packets sent and received by data channels.  In
+        particular, DTLS adds security to an underlying transport, and
+        the <code>RTCDtlsTransport</code> interface allows access to
+        information about the underlying transport and the security
+        added.
+
+        <dl class="idl" title=
+          "interface RTCDtlsTransport">
+          <dt>readonly attribute RTCIceTransport transport</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-dtlstransport-transport"><code>RTCDtlsTransport.transport</code></dfn>
+            attribute is the underlying transport that is used to send
+            and receive packets.  The underlying transport may not be
+            shared between multiple <code>RTCDtlsTransport</code>
+            objects.
+          </dd>
+
+          <dt>readonly attribute RTCDtlsTransportState state</dt>
+
+          <dd>
+            <p>The <dfn id=
+            "dom-dtlstransport-state"><code>state</code></dfn>
```

Comment:
I suggest using &lt;code id="dom-dtlstransport-state">state&lt;/code> (no &lt;dfn>) to avoid the dup id error of ReSpec.